src/main/java/net/openhft/chronicle/wire/LongValueBitSet.java

```diff
@@ -835,25 +748,11 @@ public void or(LongValueBitSet set) {
     public void xor(LongValueBitSet set) {
         int wordsInCommon = Math.min(getWordsInUse(), set.getWordsInUse());
 
-        //   if (getWordsInUse() < set.getWordsInUse()) {
-        // //      ensureCapacity(set.getWordsInUse());
-        //      wordsInUse.setValue(set.getWordsInUse());
-        //  }
-
-        // Perform logical XOR on words in common
-        for (int i = 0; i < wordsInCommon; i++) {
-            final long result;
-            result = words[i].getVolatileValue() ^ set.words[i].getVolatileValue();
```

Comment:
looking at the diff I see this code is removed, can you double check that this should be removed. All the other code in the class, I agree can be removed because its just commented out code, but this code here is different,

Reply:
This `for` loop calculates a intermediate `result` within the `for` scope, i.e., it is never used outside the scope. I don't see any side-effects from two `getVolatileValue()` calls in the calculation so I think this whole block is unused -- likely used during debugging -- and the associated tests pass. :)